code/modules/research/xenoarchaeology/tools/anomaly_suit.dm

```diff
@@ -1,14 +1,53 @@
-/obj/item/clothing/suit/space/anomaly
-	name = "Expedition suit"
-	desc = "A pressure resistant expedition suit partially capable of insulating against exotic alien energies."
-	icon_state = "anomaly_suit"
-	item_state = "anomaly_suit"
-	armor = list(melee = 25, bullet = 10, energy = 50, bomb = 0, bio = 100, rad = 100)
-	allowed = list(/obj/item/device/lighting/toggleable/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit)
-
-/obj/item/clothing/head/space/anomaly
-	name = "Expedition hood"
-	desc = "A pressure resistant expedition hood partially capable of insulating against exotic alien energies."
-	icon_state = "cespace_helmet"
-	item_state = "cespace_helmet"
-	armor = list(melee = 25, bullet = 10, energy = 50, bomb = 0, bio = 100, rad = 100)
+// hazard suit, maint version. Replaces old excavation suit.
```

Comment:
This is ported in the other Port PR

Reply:
does the other PR have the appropriate map changes for it, as well??